libs/pipelines/top_counties_pipeline.py

```diff
@@ -0,0 +1,67 @@
+from collections import namedtuple
+import logging
+
+from libs.enums import Intervention
+from libs import validate_results
+from libs import build_processed_dataset
+from libs import dataset_deployer
+
+from libs.datasets import results_schema as rc
+from libs.functions.generate_api import generate_api_for_projection
+
+logger = logging.getLogger(__name__)
+PROD_BUCKET = "data.covidactnow.org"
+
+TopCountiesPipelineProjectionResult = namedtuple(
+    "TopCountiesPipelineProjectionResult", ["key", "projection_df",]
+)
+
+TopCountiesPipelineResult = namedtuple(
+    "TopCountiesPipelineResult", ["key", "api"]
+)
+
+
+def run_projections(
+    input_file, run_validation=True
+) -> TopCountiesPipelineProjectionResult:
+    """Run the projections for the current intervention for counties 
+    in order to genereate a list of the 100 counties most affected
+
+    Args:
+        input_file: Input file to load model output results from.
+        run_validation: If true runs validation on generated shapefiles
+            and dataframes.
+
+    Returns: TopCountiesPipelineProjectionResult objects for county data.
+    """
+    # Run County level projections
+    intervention = Intervention.CURRENT
+
+    counties_key_name = f"counties.{intervention.name}"
+    # note i think build_processed_dataset should porbably be renamed? 
+    counties_df = build_processed_dataset.get_usa_by_county_with_projection_df(
+        input_file, intervention.value
+    )
+    if run_validation:
+        validate_results.validate_counties_df(counties_key_name, counties_df)
+
+    county_results = TopCountiesPipelineProjectionResult(
```

Comment:
I don't 100% know how I feel about this suggestion, but I think it would be so so cool if this returned a `DataSource` subclassed object, that was like:

```
CountyProjections(DataSource):
    class Fields(object):
        ...<all of the fields in results_schema>
```

Then in the future, we could do validation to make sure when we load something it's reasonable